mcs/class/System/System.Net.NetworkInformation/Ping.cs

```diff
@@ -243,7 +243,7 @@ public PingReply Send (IPAddress address, int timeout, byte [] buffer, PingOptio
 		private PingReply SendPrivileged (IPAddress address, int timeout, byte [] buffer, PingOptions options)
 		{
 			IPEndPoint target = new IPEndPoint (address, 0);
-			IPEndPoint client = new IPEndPoint (GetNonLoopbackIPV4 (), 0);
+			IPEndPoint client = new IPEndPoint (address, 0);
```

Comment:
I think you can keep only 1 variable with same value.
Please remove now unused `GetNonLoopbackIPV4`